pyuvdata/uvflag/uvflag.py

```diff
@@ -1678,6 +1717,10 @@ def to_baseline(
             # TODO: make this always be in the compatibility list in version 3.0
             warn_compatibility_params.append("flex_spw_id_array")
 
+        # sometimes the antenna sorting for the antenna names/numbers/positions
+        # is different. If the sets are the same, re-sort self to match uv
```

Comment:
Probably worth adding in the docstring here that `to_baseline` may re-order the antenna-related attributes, just so that no one is caught by surprise (assuming they read said docstring, of course).